fw/http.c

```diff
@@ -4720,6 +4720,44 @@ tfw_h2_hdr_sub(unsigned short hid, const TfwStr *hdr, const TfwHdrMods *h_mods)
 	return false;
 }
 
+static int
+tfw_h2_hpack_encode_trailer_headers(TfwHttpResp *resp, const TfwHdrMods *h_mods,
+			    unsigned int stream_id)
+{
+	int r;
+	unsigned int i;
+	TfwHttpTransIter *mit = &resp->mit;
+	TfwHttpHdrMap *map = mit->map;
+	TfwHttpHdrTbl *ht = resp->h_tbl;
+
+	for (i = 0; i < map->count; ++i) {
```

Comment:
It seems that its better to iterate `for (i = TFW_HTTP_HDR_RAW; i < ht->off; ++i)`

Reply:
No, it will cause a dead loop!
This code snippet is copied from [`tfw_h2_hpack_encode_headers`](https://github.com/tempesta-tech/tempesta/blob/9bc155571c17c69b0bda9f09f9c2f501c5c0d24d/fw/http.c#L4771), the only modification is for trailers.